src/client/styles/components/SubjectHeadings.scss

```diff
@@ -10,6 +10,23 @@ $darker-grey: lightgray;
     display: flex;
   }
 
+  thead {
+    .subjectHeadingsTableCell.selected {
+      color: #a83232,
```

Comment:
I think we should use a different color that doesn't match the one of subject heading labels.